standard.mk

```
@@ -68,7 +67,7 @@ gocheck: ## Lint code
 
 .PHONY: gobuild
 gobuild: #gocheck gotest ## Build binary
```

```suggestion
gobuild: gocheck gotest ## Build binary
```

Tried this locally, it works.  Recommend we go ahead and get it included and close https://github.com/openshift/cloud-ingress-operator/issues/13 and https://github.com/openshift/cloud-ingress-operator/pull/17